samples/Playground/Playground.Wasm/Playground.Wasm.csproj

```diff
@@ -1,85 +1,87 @@
+		<!-- Temporarily uncomment to generate an AOT profile https://aka.platform.uno/wasm-aot-profile -->
+		<!-- <WasmShellGenerateAOTProfile>true</WasmShellGenerateAOTProfile> -->
+	</PropertyGroup>
+	<ItemGroup>
+		<Content Include="Assets\SplashScreen.png" />
+		<Content Include="manifest.webmanifest" />
+		<Content Include="Assets\AppIcon-*.png" />
+	</ItemGroup>
+	<ItemGroup>
+		<EmbeddedResource Include="WasmCSS\Fonts.css" />
+		<EmbeddedResource Include="WasmScripts\AppManifest.js" />
+	</ItemGroup>
+	<ItemGroup>
+		<LinkerDescriptor Include="LinkerConfig.xml" />
+	</ItemGroup>
+	<ItemGroup>
+		<!--
+		This item group is required by the project template because of the
+		new SDK-Style project, otherwise some files are not added automatically.
+
+		You can safely remove this ItemGroup completely.
+		-->
+		<None Include="Program.cs" />
+		<None Include="LinkerConfig.xml" />
+		<None Include="wwwroot\web.config" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.Windows.Compatibility" Version="7.0.0" />
+		<PackageReference Include="Uno.Wasm.Bootstrap" Version="7.0.19" />
+		<PackageReference Include="Uno.Wasm.Bootstrap.DevServer" Version="7.0.19" />
+		<PackageReference Include="Uno.WinUI.WebAssembly" Version="4.7.44" />
+		<PackageReference Include="Uno.WinUI.RemoteControl" Version="4.7.44" Condition="'$(Configuration)'=='Debug'" />
+		<PackageReference Include="Uno.UI.Adapter.Microsoft.Extensions.Logging" Version="4.7.44" />
+		<PackageReference Include="Uno.Extensions.Configuration" Version="2.3.0-dev.839" />
+		<PackageReference Include="Uno.Extensions.Http" Version="2.3.0-dev.839" />
+		<PackageReference Include="Uno.Extensions.Http.Refit" Version="2.3.0-dev.839" />
+		<PackageReference Include="Uno.Extensions.Logging.WebAssembly.Console" Version="1.4.0" />
+		<PackageReference Include="Uno.Extensions.Logging.WinUI" Version="2.3.0-dev.839" />
+		<PackageReference Include="Uno.Extensions.Logging.Serilog" Version="2.3.0-dev.839" />
+		<PackageReference Include="Uno.Extensions.Serialization.Http" Version="2.3.0-dev.839" />
+		<PackageReference Include="Uno.Extensions.Serialization.Refit" Version="2.3.0-dev.839" />
+		<PackageReference Include="Uno.Material.WinUI" Version="2.5.0-dev.43" />
+		<PackageReference Include="Uno.Toolkit.WinUI.Material" Version="2.5.0-dev.53" />
+		<PackageReference Include="Uno.Toolkit.WinUI" Version="2.5.0-dev.53" />
+		<PackageReference Include="Uno.Extensions.Authentication.WinUI" Version="2.3.0-dev.839" />
+		<PackageReference Include="Uno.Extensions.Hosting.WinUI" Version="2.3.0-dev.839" />
+		<PackageReference Include="Uno.Extensions.Localization.WinUI" Version="2.3.0-dev.839" />
+		<PackageReference Include="Uno.Extensions.Navigation.Toolkit.WinUI" Version="2.3.0-dev.839" />
+		<PackageReference Include="Uno.Extensions.Navigation.WinUI" Version="2.3.0-dev.839" />
+		<PackageReference Include="Uno.Extensions.Reactive.WinUI" Version="2.3.0-dev.839" />

```

Comment:
Be sure to replace all of the Package References from the template with Project References for the Playground